tests/edge-cases/point_add-05.txt

```diff
@@ -0,0 +1,5 @@
+brun '(point_add (q (1 2)) (q (1 2)))'
+FAIL: point_add expects blob, got ff01ff0280: from_bytes(): incompatible function arguments. The following argument types are supported:
+    1. (self: buffer) -> blspy.G1Element
+
+Invoked with: None ((q 2) (q 2))
```

Comment:
I'm not sure about this one. Is this correct? the output doesn't follow the pattern used by other operator failures, which I've learned sometimes indicate the wrong kind of exception is thrown (i.e. an unexpected exception)

Reply:
Yeah, probably worth checking the pair/atom type and raising a different message prior to trying to evaluate.

Reply:
done, please review